src/pages/[guild]/collect/[chain]/[address].tsx

```diff
@@ -290,7 +290,11 @@ const getStaticProps = async ({ params }) => {
       fallbackImage: nftGuildReward.platformGuildData.imageUrl,
       // Pre-populating the public guild & requirements caches
       fallback: {
-        [guildPageEndpoint]: publicGuild,
+        [guildPageEndpoint]: {
+          ...publicGuild,
+          // An NFT is always public (it can be found on the blockchain), so we can add the guildPlatform to the public guild cache
+          guildPlatforms: [...publicGuild.guildPlatforms, nftGuildReward],
```

Comment:
Won't this duplicate it, in the case, when `nftGuildReward` is already public?

Reply:
Uh-oh, good point, added some extra logic in https://github.com/guildxyz/guild.xyz/pull/1276/commits/1f9d89080fb1d7f012a2061ac7053bc7932c94fd, let me know if it looks good to you!

Reply:
Thanks for the change! ✨